apps/site/src/app/(main)/apply/sections/Title/Title.tsx

```diff
@@ -1,15 +1,11 @@
-import styles from "./Title.module.scss";
-
 export default function Title() {
 	return (
 		<>
-			<h1
-				className={`${styles.title} font-display text-[#fffce2] text-8xl text-center max-[500px]:text-7xl`}
-			>
-				Apply
+			<h1 className="text-[var(--color-white)] text-center font-display font-bold text-3xl md:text-5xl ">
+				Apply as a Hacker
```

Comment:
Issue: since I'm assuming this `Title` component is to be reused, pass in the application type (Hacker, Mentor, Volunteer) as a prop rather than hard coding "Hacker"